qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -355,3 +355,27 @@ def _to_sparse_label(label):
         return " ".join(
             filter(None, (label_transformation[c].format(i=i) for i, c in enumerate(label)))
         )
+
+    @classmethod
```

Comment:
Why do we need these class methods? (same comment for `VibrationalSpinOp`)

Reply:
It looks like they could be static but `@classmethod `forces them to still be part of the class and helps to avoid passing unnecessary `self`.

Reply:
The addition of these was motivated by https://github.com/Qiskit/qiskit-nature/pull/220/commits/850267550d686cd49813aa432d491609dda08c7c
That commit showed that allowing an empty operator initialization is not straight forward. Having a `zero` and `one` operator available on the class itself is pure convenience (which in the future we may use in other places, too)

Reply:
Personally, I don't like having multiple ways to do one thing, but I'll leave it to you. And, opflow is not planning to support these methods, so it will be inconsistent, but it might be a convenient method for second quantized op. 
Finally, this initialization can be applied as follows:
```python
FermionicOp(("", 0), register_length=register_length)
FermionicOp((f"I_{register_length - 1}", 0))
0 * FermionicOp(f"I_{register_length - 1}")
```

Reply:
I support having the concept and methods for zero and one. For creating and also predicates. I think initializing a zero and one is not straightforward; neither programmatically, nor conceptually. What exactly should count as zero?  I was struggling today with how to construct a zero `SparsePauliOp`. I sort of gave up and thought about opening an issue asking about it, and suggesting methods for zero and one.